lib/roll/app_builder.rb

```diff
@@ -79,11 +79,11 @@ def configure_generators
     end
 
     def configure_mailers_preview_path
-      config = <<-RUBY
+      config = <<-'RUBY'
```

Comment:
Adding ' disable interpolation, which is needed here (`Rails.root` shouldn't be interpolated before app execution)